core/src/main/java/io/cucumber/core/options/RuntimeOptionsBuilder.java

```diff
@@ -32,8 +31,7 @@
     private PickleOrder parsedPickleOrder = null;
     private Integer parsedCount = null;
     private Class<? extends ObjectFactory> parsedObjectFactoryClass = null;
-    private boolean addDefaultSummaryPrinterIfAbsent;
-    private boolean addDefaultFormatterIfAbsent;
+    private boolean addSummary = true;
```

Comment:
I would consider printing a summary a sensible default only for the CLI.  So not enabling the summary printer by default and enabling it only in the CLI would be better behaviour. It would also avoid the need to explicitly disable this plugin in various tests again which simplifies things further.

Reply:
Fixed in https://github.com/cucumber/cucumber-jvm/pull/2197/commits/d0ba0bbe774b2b6eee7c44156d4f5827806b4fdd